bitmovin-analytics-datasource/src/components/ConfigEditor.tsx

```diff
@@ -8,6 +8,12 @@ interface Props extends DataSourcePluginOptionsEditorProps<MyDataSourceOptions>
 export function ConfigEditor(props: Props) {
   const { onOptionsChange, options } = props;
 
+  useEffect(() => {
```

Comment:
Can we add a quick comment here what the `useEffect` is aiming for?
I saw it in the dashboard and i think it's a good thing to have